tslint.json

```diff
@@ -1,63 +1,51 @@
 {
+  "defaultSeverity": "error",
+  "extends": [
+    "tslint:all",
+    "tslint-config-prettier"
+  ],
   "rules": {
-    "class-name": true,
-    "comment-format": [
-      true,
-      "check-space"
-    ],
-    "indent": [
-      true,
-      "spaces",
-      2
-    ],
-    "one-export": false,
-    "no-console": true,
-    "no-debugger": true,
-    "no-eval": true,
-    "no-internal-module": true,
-    "no-trailing-whitespace": true,
-    "no-unsafe-finally": true,
-    "no-var-keyword": true,
-    "one-line": [
-      true,
-      "check-open-brace",
-      "check-whitespace"
-    ],
-    "quotemark": [
-      true,
-      "single",
-      "avoid-escape"
-    ],
-    "semicolon": [
-      true,
-      "never"
-    ],
-    "triple-equals": [
-      true,
-      "allow-null-check"
-    ],
-    "typedef-whitespace": [
+    "completed-docs": [
       true,
       {
-        "call-signature": "nospace",
-        "index-signature": "nospace",
-        "parameter": "nospace",
-        "property-declaration": "nospace",
-        "variable-declaration": "nospace"
+        "interfaces": {
+          "visibilities": [
+            "exported"
+          ]
+        },
+        "classes": {
+          "visibilities": [
+            "exported"
```

Comment:
something weird happening here:
- technically it should be written as `visibilities: "exported"` since it's an enum
- but for some reason, it only works when written like this.

Reply:
https://palantir.github.io/tslint/rules/completed-docs#schema